MIT-LICENSE

```diff
@@ -0,0 +1,20 @@
+Copyright 2015 Jerimiah Milton
```

Comment:
The license should be an Apache 2 license, and it should be "2015 Booz Allen Hamilton"